draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1371,6 +1400,37 @@ Section Header
 
             <t>Example: '0'.</t>
 
+            <t hangText="epb_packetid:"><vspace blankLines="0"/>The
+            epb_packetid option is a 64-bit unsigned integer that uniquely
+            identifies the packet. If the same packet is seen by multiple
+            interfaces and there is a way for the capture application to
+            correlated them, the same epb_packetid value must be used. An
+            example could be where an XDP program on ingress captures a packet,
+            which gets modified and then exits the XDP program. In this case,
+            two packets are in the capture file, which are not identical but
+            the epb_packetid can be used correlate them.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_queue:"><vspace blankLines="0"/>The epb_queue
+            option is a 32-bit unsigned integer that identifies on which queue
+            of the interface the specific packet was received.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_verdict:"><vspace blankLines="0"/>The epb_verdict
```

Comment:
What does "verdict" mean? I assume it means firewall/ACL activity like "drop", "reject", "accept"? Not all readers might be familiar with the term, so it would be good to explain it just a bit :-)

Reply:
Added some additional explanation

Reply:
By the way, the Linux_eBPF_TC and Linux_eBPF_XDP verdict types appear to be integral values; if so, are they in the byte order of the host writing the file (so that a little-endian machine will write them as little-endian numbers and a big-endian machine will write them as big-endian numbers)?

And is the hardware verdict type an opaque blob of bytes?

Reply:
(The byte order question also applies to the packet ID and queue options, with presumably the same answer.)

Reply:
The Linux_eBPF_TC and Linux_eBPF_XDP are stored a uint64 and hence the same as all other metadata in the block, so in host order as explained in the "Endianness" section. 

You are right about the hardware verdict type, this is an opaque blob of bytes.

Packet and queue ID are the same as any other option, for example, epb_dropcount, which are stored as explained in the "Endianness" section.